konrad/ozone.py

```diff
@@ -239,6 +238,10 @@ def simotrostra_profile(self, o3, atmosphere, timestep, zenith):
         transport_ox = self.ozone_transport(o3, z)
         do3dt = source - sink_ox - sink_nox + transport_ox - sink_hox
         o3_new = o3 + do3dt*timestep
+
+        # prevent concentrations getting too low - set tropospheric value
+        o3_new[o3_new < 4 * 10**-8] = 4 * 10**-8
```

Comment:
One could use the ``clip()`` method:
```python
o3_new.clip(min=4e-8, out=o3_new)
```
It is slightly faster (although I agree, that this does not matter here) and more concise in its syntax.
But I am fine with the current implementation, choose the one you find more readable 😉